diff --git a/cop-o-matic.py b/cop-o-matic.py
index 96eacb0..d9791c3 100755
--- a/cop-o-matic.py
+++ b/cop-o-matic.py
@@ -30,6 +30,11 @@ class Change:
         self.current_revision = CurrentRevision(commit_message, name, email)
         self.url = lambda: hash
 
+def kickstart_git():
+    subprocess.check_output(["wget", "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/clone.bundle", "-O", "/tmp/clone.bundle"])
+    subprocess.check_output(["git", "remote", "add", "bundle", "/tmp/clone.bundle"])
+    subprocess.check_output(["git", "fetch", "bundle"])
+
 
 def do_review(project, change, verbose):
     c = change
@@ -59,6 +64,11 @@ def do_review(project, change, verbose):
         print("Reviewer not found")
         return 0
 
+    try:
+        kickstart_git()
+    except:
+        pass
+
     reviewer.gerrit_patch = c.patch
     reviewer.change = c
     review = reviewer.review_patch()
